docs/docs-components/SkipToContent.js

```diff
@@ -0,0 +1,55 @@
+// @flow strict
+import { useState, useCallback, useEffect, type Node } from 'react';
+import { Box, Text, TapArea } from 'gestalt';
+import { SKIP_TO_CONTENT_ZINDEX } from './z-indices.js';
+
+/**
+ * Skip to content allows you to skip directly to the content on the page
+ *
+ * Note: This TapArea will only show when people are tabbing through (for accessibility purposes)
+ */
+export default function SkipToContent(): Node {
+  const [focused, setFocused] = useState(false);
+  const [mainContent, setMainContent] = useState(null);
+
+  const handleTabIndex = useCallback(() => {
+    mainContent?.removeAttribute('tabindex');
+    mainContent?.removeEventListener('blur', handleTabIndex);
+    mainContent?.removeEventListener('focusout', handleTabIndex);
+  }, [mainContent]);
+
+  useEffect(() => {
+    if (!mainContent) setMainContent(document.querySelector('[role="main"]'));
+
+    return () => {
+      // remove event listener in case we unmount component without calling handleTabIndex
+      mainContent?.removeEventListener('blur', handleTabIndex);
+      mainContent?.removeEventListener('focusout', handleTabIndex);
+    };
+  }, [mainContent, handleTabIndex]);
+
+  return (
+    <TapArea
+      onBlur={() => setFocused(false)}
+      onFocus={() => setFocused(true)}
```

Comment:
Canw e underling or add a focus ring tot he text when it's focused? It's a bit unclear atm that the Skip to Content button has the focus

Reply:
Yeah, let me see. I coudn't figure out why it wold not get the focus ring....

Reply:
I underlined it as I can't figure out why it's not displaying the focus ring.